### PR TITLE
Fix spacing in vm driver selection log

### LINKED ISSUE #6439: Extraneous space in "Automatically selected" line
Notice the space before 'kvm2'. I suspect this has to do with the experimental text change.

```
😄  minikube v1.7.0-beta.1 on Debian rodete
✨  Automatically selected the  'kvm2' driver (alternates: [none docker])
🔥  Creating kvm2 VM (CPUs=2, Memory=2000MB, Disk=20000MB) ...
🐳  Preparing Kubernetes v1.17.2 on Docker '19.03.5' ...
```

### PR DESCRIPTION
**Pull request by priyawadhwa**
Fixes #6439 

Output is now:

```
$ minikube start
😄  minikube v1.7.0-beta.1 on Debian rodete
✨  Automatically selected the 'kvm2' driver (alternates: [none docker])
```

### CODE REVIEW

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -596,14 +596,14 @@ func selectDriver(existing *config.MachineConfig) string {
 		if pick.Priority == registry.Experimental {
 			exp = "experimental"
```
Comment (sharifelgamal) [inline cmd/minikube/cmd/start.go]: this needs to be exp = "experimental "

Comment (priyawadhwa) [reply cmd/minikube/cmd/start.go]: thanks, done

**Comment by medyagh** [comment]
Do you mind sharing the output in the pr description to compare ?

**Comment by priyawadhwa** [comment]
@medyagh done

**Pull request merged**